web/src/bound-edit-activity.js

```diff
@@ -125,7 +127,7 @@ const mapDispatchToProps = dispatch => ({
     if (file) {
       if (file.includes('/lib/') || file.includes('/data/') || file.includes('/audio/')) {
         console.log('files under /lib/, /data/, and /audio/ cannot be run as a script');
-        return undefined;
+        return;
```

Comment:
Eslint is expecting consistent return values, so we either need to explicitly return `undefined` in all code paths, or return `void` (which should end up being `undefined` anyway).